### PR TITLE
output graph generated as a html file.

### PR DESCRIPTION
**Pull request by Harikarthyk**
Now an HTML file will be generated for the D3 graph.

This modification has now been completed in **order to add tooltips, and other DOM-related changes can now be implemented.**

### CODE REVIEW

File: services/fastapi/routers/createsvg.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ async def execute_createsvg(request: Request, response: Response, body: CodeRequ
     # generate unique request id
     requestid = f'{datetime.timestamp(datetime.now())}.{str(random.randint(1, 100000))}'
     # get new name for a new script
-    new_output_name = f'req-{requestid}.svg'
+    new_output_name = f'req-{requestid}.html'
```
Comment (wildone) [inline services/fastapi/routers/createsvg.py:50]: is this a default behaviour or an option?

**Review by wildone — CHANGES_REQUESTED** [review]
so you can not generate SVG anymore?

**Comment by wildone** [comment]
this must not be right. 
1. that router is called create svg and its for creating SVGs so that they can be embedded into places. 
2. should be just adding a new router for this.
3. is generating svg not needed at all?

File: services/fastapi/routers/createsvg.py (hunk 1 above)
Comment (Harikarthyk) [reply services/fastapi/routers/createsvg.py:50]: Right now, this route is used for generating a d3 graph (the output viz), and for the tooltip algorithm, the output thing should be in HTML format, so I have changed the things from.svg to.html.

**Pull request closed without merging**